config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -115,11 +117,3 @@ presets:
   # Increase size of default subnetworks to enable large clusters.
   - name: ENABLE_BIG_CLUSTER_SUBNETS
     value: "true"
-# Create a project k8s-jenkins-scalability-head and move this test there
-- labels:
-    preset-e2e-gci-gce-scalability: "true"
```

Comment:
There is one more place, where this preset is used - config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

Can you remove there also?